### ISSUE TITLE
New CDS

### ISSUE THREAD

**Issue by Peter9192** [WAM2layers/WAM2layers#409]
As reported by @P-Myers, ERA5 data from the new CDS portal is slightly different. Beta data has a few variable name differences compared to the previous data versions, namely "time" is now "valid_time" and "level" is now "model_level".

It would be nice to support this data source out of the box; we could have 

```
wam2layers preprocess era5  # This would be the old CDS (for backward compatibility)
wam2layers preprocess era5-newcds  # This would be using data from the new CDS
```

_Originally posted by @Peter9192 in https://github.com/WAM2layers/WAM2layers/discussions/408#discussioncomment-10771554_

**Comment by BSchilperoort** [comment]
I feel like it might be better to detect which CDS version was used by looking at the name of the time coordinate (or possibly the netCDF attributes), and not bother the end users with this.

Finishing up #401 could also be a stopgap solution while the CDS is still unstable (the new grib->netCDF converter is unstable and is likely to change still, and the cads-api-client is as well despite it being a >1.0 version :roll_eyes: )

**Comment by ruudvdent** [comment]
Thanks Yiying. This can be closed per #429
